concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
+
+inputs:
+  - name: cf-manifest
+  - name: bosh-secrets
+run:
+  path: sh
+  args:
+  - -c
+  - -e
+  - |
+    bosh_password=$(awk '/bosh_admin_password/ { print $2 }' bosh-secrets/bosh-secrets.yml)
+    bosh -t 10.0.0.6 login admin "${bosh_password}"
```

Comment:
We have this pattern - get password, set target, login - in quite a few places. Do you think it's worth refactoring into a separate script?
```
➜  paas-cf git:(master) ✗ ack 'bosh .* target'
concourse/pipelines/deploy-cloudfoundry.yml
496:              bosh -u admin -p "${bosh_password}" target https://10.0.0.6:25555
526:            bosh -u admin -p "${bosh_password}" target https://10.0.0.6:25555
552:            bosh -u admin -p "${bosh_password}" target https://10.0.0.6:25555

concourse/scripts/bosh_ensure_uploaded.sh
6:bosh -u admin -p "${BOSH_PASSWORD}" target https://10.0.0.6:25555 >/dev/null
```